src/Propulsion.MessageDb/MessageDbSource.fs

```diff
@@ -71,7 +71,7 @@ module Internal =
 
     let internal readBatch batchSize (store : MessageDbCategoryClient) (category, pos, ct) : Task<Core.Batch<_>> =
         let positionInclusive = Position.toInt64 pos
-        store.ReadCategoryMessages(category, positionInclusive, batchSize, ct)
+        store.ReadCategoryMessages(category, positionInclusive + 1L, batchSize, ct)
```

Comment:
For `Propulsion.Feed`, a feed Position is generally 0-based, with a >= semantic (empahsis on the second aspect - ultimately the value is not actually semantically meaningful to the infra)
In the naming and impl here, it's following the same concept (`Inclusive` etc)
Hence I think the +1 should be shifted to L41 (and also needs to be addressed in readTailPositionForTranche)

This will be reflected in log emissions too.

(The other reason for letting the read logic take care of computing the next position is that ESDB and SSS have a 'NextPosition', and e.g. a filter might mean that the actual next read position can be far beyond the most recently read event)